### PR TITLE
Added some helpful Quick Help comments

### PR DESCRIPTION
**Pull request by matt-lebl**
I was a bit confused as to the parameters used in the test app that's bundled along with Hermes, and when I tried Quick Help on some of the attributes it didn't return anything useful. I went ahead and added some for you guys. It works when you Option-click on the attributes within `ViewController.swift`.

Hope this is helpful, and if not, tell me so I can make it better!

### CODE REVIEW

File: Hermes/HermesNotification.swift [hunk 1]
```diff
@@ -19,13 +20,27 @@ public class HermesNotification: NSObject {
...
+  /// Text, but with fancy attributes instead of a regular `String`
   public var attributedText: NSAttributedString?
+  
+  /// The color of the bottom bar on the notification
   public var color: UIColor?
+  
+  /// The image to be displayed along with the notification
   public var image: UIImage?
+  
+  /// The URL to the image
```
Comment (matt-lebl) [inline Hermes/HermesNotification.swift:33]: I know this is crappy. I couldn't find the purpose of this attribute.

File: Hermes/HermesNotification.swift [hunk 2]
```diff
@@ -19,13 +20,27 @@ public class HermesNotification: NSObject {
...
   public var imageURL: NSURL?
   public var tag: String?
```
Comment (matt-lebl) [inline Hermes/HermesNotification.swift:35]: This one I also couldn't find.

**Comment by matt-lebl** [comment]
Awh, I'll bet you guys are at Camp Imgur... enjoy yourselves!

**Comment by jpmcglone** [comment]
Haha, yes we were at Camp Imgur, and it was so much fun!  Thanks for the helpful comments.  Will merge it in and make sure to do more cleanup / commenting moving forward.  And sorry about the incomplete feature :( I meant to get imageURL working (and still intend to w/ SDWebImage)

File: Hermes/HermesNotification.swift (hunk 1 above)
Comment (jpmcglone) [inline Hermes/HermesNotification.swift:33]: Sorry, I never completed this feature. I should have removed it, but I'll throw in an issue for now.

File: Hermes/HermesNotification.swift (hunk 2 above)
Comment (jpmcglone) [inline Hermes/HermesNotification.swift:35]: tag? It's not used in the demo. It's completely optional, and is sometimes useful if you have specially designed notifications.

**Comment by matt-lebl** [comment]
Sounds great. :+1:

**Pull request merged**